.github/workflows/interrogate.js

```diff
@@ -16,6 +16,11 @@ const {execSync} = require('child_process');
 execSync('git fetch origin master');
 const baseRef = process.env.GITHUB_BASE_REF;
 let status;
+
+execSync('git fetch origin master');
```

Comment:
It will break once github changes the default branch from `master` to something else.

Reply:
we re no longer using a backed in vlue.